### ISSUE TITLE
Error with comments

### ISSUE THREAD

**Issue by zuf** [less/less.ruby#70]
Strange error with this code:

```
div
/* comment */
{
  color: red;
}
```

If remove comment line all works fine.

**Comment by andrehjr** [comment]
I think I got it right on 
http://github.com/andrehjr/less/commit/79d681d610b9ced1c1f9e6ae0e1782b70cd68fea

Please take a look ;)

**Comment by samalone** [comment]
Yes, this was a big issue with the Zen theme in Drupal, which I was converting to less. I'll have to try out the changes from andrehjr.

**Comment by cloudhead** [comment]
I'm pulling this in, thanks.
